src/netius/test/extra/proxy_c.py

```
@@ -0,0 +1,567 @@
+        self.assertTrue("myapp.example.com" in server.alias)
+
+        # simulates second rebuild with the service removed,
+        # the suffix alias should be properly cleaned up
+        server._build_consul([])
+
+        self.assertTrue("myapp" not in server.hosts)
+        self.assertTrue("myapp.example.com" not in server.alias)
+        server.cleanup()
+
+    def test_consul_fetch(self):
+        if mock == None:
```

Ruff (0.14.14)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)